go/utils/setup.go

```diff
@@ -17,12 +22,42 @@ const (
 
 var Nodes map[string]node
 var Network string
+var Testnet string
+
+var configFilePath = "../../kubernetes/helm/functional-testing/values/test.yaml"
```

Comment:
I do not want to read the config file raw here, but instead read the env vars that are passed to the container at runtime (environmental info) - if needed.
Thie values/*.yaml files are meant for being resolved when generating the kubernetes templates, look at: https://github.com/centrifuge/functional-testing/blob/8a9d7916225b35a6046d72b3990e6e9dc0220612/kubernetes/helm/functional-testing/chart/functionalTestingChart/templates/deployment.yaml#L13

In `setup.go` we read: `os.Getenv("NODES")` we will just add another one for paymentObligation: `os.Getenv("CONTRACTS_PAYMENT_OBLIGATION")` and then the code will use that contract address.

Reply:
Let's say we have two values file one for russianhill and the other one for bernalheights.
how would I define it in the deployment.yaml which one it should take?
```
value: {{ $.Values.nodes }} 
```

Reply:
That is defined in spinnaker depending on the job/pipeline per environment